lib/wolfe/cleanup.rb

```diff
@@ -64,21 +65,16 @@ def cleanup( config )
         end
       end
 
-      def calculate_monthly_date_for_backup_to_keep(one_per_month_timespan)
-        timespan_digit = one_per_month_timespan.split('.')[0].to_i
-        timespan_word = one_per_month_timespan.split('.')[1].to_sym
+      def calculate_monthly_date(one_per_month_timespan)
```

Comment:
I wasn't sure about this method. Should it be defined in `Cleanup` or `TimespanFromConfiguration` class?

Reply:
I think Cleanup is fine.